src/ripple/app/rdb/backend/impl/SQLiteDatabase.cpp

```diff
@@ -1604,50 +1609,50 @@ RelationalDBInterfaceSqliteImp::getTransaction(
 }
 
 bool
-RelationalDBInterfaceSqliteImp::ledgerDbHasSpace(Config const& config)
+SQLiteDatabaseImp::ledgerDbHasSpace(Config const& config)
```

Comment:
Ledger doesn't mean node store, so it searches either ledger database. BTW, this function isn't used.

Reply:
Since it's unused and untested I'd be inclined to remove it.  It's serving no useful purpose.  But I won't insist on you removing it right now, since the problem predates you.

Reply:
Ok, we can defer for now